yourgov/components/FormMobileFoodVendorPermit/FormState.ts

```diff
@@ -3,29 +3,25 @@ import { DeepPartial } from '../../lib/types';
 import {
 	defaultFormState as defaultTaskFormState,
 	type FormState as TaskFormState,
-} from './task/FormState';
-import { type StepNumber } from './task/FormProvider';
+} from './steps/FormState';
+import { type StepNumber } from './steps/FormProvider';
 
 export type Completion = {
 	completed: boolean;
 	started?: boolean;
 };
 
-type TaskCompletion = Completion & {
-	completedRecently: boolean;
-};
-
 export type FormState = {
-	task: Partial<TaskFormState> & TaskCompletion;
+	id: string;
+	lastUpdated: number;
+	steps: Partial<TaskFormState>;
```

Comment:
Added an `id` and `lastUpdated` to support the permits page. Renamed `task` to `steps`

Reply:
Should we be renaming `TaskFormState`?

Reply:
Oh I see, it's just the key so far.

Reply:
Yep, I probably should update that

Reply:
@ChrisLaneAU Just pushed [another commit](https://github.com/agriculturegovau/agds-next/pull/1843/commits/f153439a9aadbacfb35b85fc614f375602eddfc6) to remove references to "task"